suite/tests/client-interface/cleancall-opt-shared.h

```diff
@@ -781,3 +781,43 @@ codegen_bbcount(void *dc)
     codegen_epilogue(dc, ilist);
     return ilist;
 }
+
+/* Clobber aflags.  Inliner must ensure they are restored.
```

Comment:
Please update the comment: Clean call optimizations must ensure they are restored.